### PR TITLE
MRD-4523 -  Enter details for recall type page

### PR DESCRIPTION
**Pull request by jonnywyatt**

### CODE REVIEW

File: e2e_tests/integration/caseSummary.feature [hunk 1]
```diff
@@ -11,8 +11,8 @@ Feature: Case summary
     And Maria views the Contact history page
     And Maria filters contacts by date range
     And Maria starts a new recommendation
-    And Maria recommends a fixed term recall
-    And Maria selects a custody status
-    Then Maria sees a confirmation page
-    And Maria downloads the Part A
-    And Maria updates the recommendation
\ No newline at end of file
+#    And Maria recommends a fixed term recall
```
Comment (jackmaloney) [inline e2e_tests/integration/caseSummary.feature:14]: You mean to comment this out?

File: integration_tests/accessibility/accessibility.spec.ts [hunk 2]
```diff
@@ -20,8 +20,8 @@ const urls = [
     url: `${routeUrls.cases}/123/contact-history?dateFrom-day=13&dateFrom-month=24&dateFrom-year=22&dateTo-day=14&dateTo-month=20&dateTo-year=22`,
   },
   // recommendation flow
-  { url: `${routeUrls.recommendations}/456/recall-type` },
-  { url: `${routeUrls.recommendations}/456/recall-type`, validationError: true },
+  // { url: `${routeUrls.recommendations}/456/recall-type` },
```
Comment (jackmaloney) [inline integration_tests/accessibility/accessibility.spec.ts:23]: You mean to comment this out?

File: e2e_tests/integration/caseSummary.feature (hunk 1 above)
Comment (jonnywyatt) [reply e2e_tests/integration/caseSummary.feature:14]: Yes, the E2E will fail until your PR is in as well, then I'll re-enable them

File: integration_tests/accessibility/accessibility.spec.ts (hunk 2 above)
Comment (jonnywyatt) [reply integration_tests/accessibility/accessibility.spec.ts:23]: Yes, will be reintroduced when E2E working, see above

File: server/controllers/recommendations/postRecommendationForm.test.ts [hunk 3]
```diff
@@ -39,19 +40,22 @@ describe('postRecommendationForm', () => {
...
     await postRecommendationForm(req, res)
     expect(res.redirect).toHaveBeenCalledWith(303, currentPageUrl)
     expect(req.session.errors).toEqual([
       {
-        href: '#recallType',
-        name: 'recallType',
-        text: 'Select a recommendation',
-        errorId: 'noRecallTypeSelected',
+        errorId: 'missingRecallTypeDetail',
+        fieldText: 'Enter more detail',
```
Comment (jackmaloney) [inline server/controllers/recommendations/postRecommendationForm.test.ts:52]: Is this message correct? The ACs in MRD-460 suggest something different. Can you update the ticket if appropriate?

Comment (jonnywyatt) [reply server/controllers/recommendations/postRecommendationForm.test.ts:52]: yep will update ticket. As mentioned, this needs approving by the content designer anyway

**Review by jackmaloney — APPROVED** [review]

**Pull request merged**